ONSdigital/census-rh-ui
# CR-1438 Combine pages

Combine 'england' and 'wales' pages into 'england-and-wales' page in 'start' journey

Signed-off-by: Paul-Joel <paul.joel@ons.gov.uk>

# Motivation and Context
Start 'address not in northern ireland' 'england' and 'wales' pages are to be combined

# What has changed
Combined England and Wales pages
Removed redundant page
Updated unit tests

Change is not required in the fulfilments journeys, only the UAC authentication journey

No Cucumber updates required

# How to test?
Enter a 'E' or 'W' region UAC on the /ni/start/ page, and check that the resulting page says 'You have entered an access code for the census in England and Wales.'

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1438
